ocaml/xapi/helpers.ml

```diff
@@ -514,8 +514,11 @@ let boot_method_of_vm ~__context ~vm =
     NB that just because a VM's current configuration looks like HVM doesn't imply it
     actually booted that way; you must check the boot_record to be sure *)
 let is_hvm ~__context (x: API.vM_t) = 
```

Comment:
Can't we just replace this with:
```ocaml
let is_hvm (x: API.vM_t) = x.API.vM_HVM_boot_policy <> ""
```

The additional `~__context` parameter only came in recently, it can go if we don't need it.

@jonludlam what do you think?